app/_how-tos/configure-google-cloud-secret-as-a-vault-backend.md

```diff
@@ -45,6 +57,78 @@ cleanup:
+
+```sh
+export GCP_SERVICE_ACCOUNT=$(cat gcp-project-c61f2411f321.json)
+```
+
+## 2. Configure {{site.base_gateway}} to trust Google Cloud API SSL certificates
+
+Configure `lua_ssl_trusted_certificate=system` in your `kong.conf` file [LINK!!!]. This ensures that the SSL certificates that the Google Cloud API uses can be trusted by {{site.base_gateway}}. You can configure this directly in your `kong.conf`, or use [environment variables](/deck/environment-variables/):
+
+```sh
+export KONG_LUA_SSL_TRUSTED_CERTIFICATE='system'
+```
+
+## 3. Configure Secret Manager as a vault with the Vault entity
+
+To enable Secret Manager as your vault in {{site.base_gateway}}, you can use the [Vault entity](/gateway/entities/vault).
+
+For the sake of the tutorial, we aren't separating the Vault decK configuration from the other entities, like Consumers and Services. However, if you're implementing a Vault configuration in production and have a large organization with many teams, we recommend splitting the Vault configuration from other entities with decK. For more information, see [Declarative configuration (decK) best practices for Vaults](/gateway/entities/vault/#declarative-configuration-deck-best-practices-for-vaults).
+
+Add the following content to `kong.yaml` to create a Secret Manager Vault:
+
+{% entity_examples %}
+entities:
+  vaults:
+    - config:
+        project_id: test-gateway-vault
+      description: Stored secrets in Secret Manager
+      name: gcp
+{% endentity_examples %}
+
+## 4. Reference the secret from your Vault in {{site.base_gateway}} configuration
+
+Now that Secret Manager is configured as your vault, you can reference secrets stored in that vault in configuration. In this tutorial, you'll be referencing the API key you set previously and using it to generate an answer to a question using the [AI Proxy plugin](/plugins/ai-proxy/). To reference a secret, you use the provider name from your Vault config, the name of the secret, and the property in the secret you want to use.
+
+Add the following content to `kong.yaml` to reference the Vault secret as your bearer token:
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: ai-proxy
+      route: example-route
+      config:
+        route_type: llm/v1/chat
+        auth:
+          header_name: Authorization
+          header_value: '{vault://gcp-sm-vault/test-secret/key}'
+        model:
+          provider: mistral
+          name: mistral-tiny
+          options:
+            mistral_format: openai
+            upstream_url: https://api.mistral.ai/v1/chat/completions
+{% endentity_examples %}
+
+## 5. Apply configuration
+
+{% include how-tos/steps/apply_config.md %}
+
+## 6. Validate
+
+To verify that {{site.base_gateway}} can pull the secrets from Secret Manager, you can use the AI Proxy plugin to confirm that the plugin is using the correct API key when a request is made:
+
+```
```

Comment:
you shoulduse the `{% validation request-check %}` block for this type of curl commands, similar to [this one](https://github.com/Kong/developer.konghq.com/blob/main/app/_includes/how-tos/steps/ai-proxy-validate.md?plain=1)